l10n_be_cooperator_portal_national_number/controllers/portal.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from unittest import mock
+
+from odoo import _
+from odoo.http import request, route
+
+from odoo.addons.account.controllers.portal import PortalAccount
+
+
+class CooperatorPortal(PortalAccount):
+    def __init__(self, *args, **kwargs):
+        super().__init__()
+        if "OPTIONAL_BILLING_FIELDS" not in vars(self):
+            self.OPTIONAL_BILLING_FIELDS = PortalAccount.OPTIONAL_BILLING_FIELDS.copy()
```

Comment:
why is this called `BILLING`? is this related to billing? this is about personal information, right?

Reply:
no idea! code copied from `cooperator_portal`, which you wrote ;) added comment